src/sgame/sg_active.cpp

```diff
@@ -2257,6 +2257,20 @@ void ClientThink( int clientNum )
 	{
 		ClientThink_real( ent );
 	}
+
```

Comment:
Can we instead fix the root cause of why opening the menu in ClientBegin doesn't work?

Reply:
The reason that this doesn't work in ClientBegin doesn't work is that when the client connection state becomes `active`, we hide all the windows. We should fix that.

Reply:
I'd also prefer to fix the root cause, but I don't know much the cgame side of the codebase. As I said, I only stole the original commit and applied a cleanup on the ugly switch() case, because I think this feature is highly desirable.